content/security-humanops.rst

```diff
@@ -0,0 +1,231 @@
+
+Consider what services are facing the Internet.
+
+Consider what information is facing the Internet.
+
+When posting reviews or comments, are there certain details that need to be taken out to sanitize the review or comment?  
+
+Is this public information going to help with someone's enumeration activities? Are there sensitive files, passwords, private key info publicly facing? 
+
+
+Customers
+=========
+
+We work with admins that are already aware of security practices - like not leaving sensitive information in logs.
+
+However we work with a number of customer bases and budgets.
+
+
+Olark
+-----
+
+We use Olark web chat. If talking to a customer, please verify who they are and their installation.  The other alternative is to move the discussion to email.
+
+
+Improving Documentation
+-----------------------
+
+If there is a customer having trouble securing their setup and they have read the documentation, ask what they are trying to do.  Then detect what portion of the documentation can be improved and create a ticket for this.
+
+If a customer has gone to support for this issue and it is not an 'edge case', then chances are high that others have had this issue before.
+
+
+Testing / validating third party apps / software for use
+========================================================
+
+There are some instances when we need to set up test accounts - for example to trial a ticketing software, an online web software and so on. When doing a test, try to test using your own individual Pro:Atria email as it helps solve 'noise'.
+
+Take note of what third party services are opened to do testing in the internal wiki.  
+
+If the third party service is no longer being used, or when testing is finished, delete all instances of any irrelevant tests/accounts and unused services.  Then update the wiki with your findings.
+
+If an app is in active use with customers / within the team during the validating period, ensure that whitelisting takes place to help validate/verify that it is an Pro:Atria-used third party product.
+
+
+Traveling, use of multiple devices etc
+=======================================
+
+If you are planning to travel while working, there are additional precautions faced being on the move.  Anything from risk of using hostile networks, to risk of device being stolen/tampered with.  
+
+When in a public space, use a VPN to secure the connection or consider tethering to your phone.
+
+If possible, connect via wired instead of wireless connection.
+
+There is no request, customer or team, so urgent that you need to connect to a potentially hostile public network then and there to do the task.  Mention this and continue to do the task when you are in a better network.
+
+When faced with "I'm overseas and don't have access to..." or similar type of request, ask to use a secondary channel (like a phone call) to continue with the request.  In this case team member contact details must always be kept up to date in the wiki.
```

Comment:
Ditto with https://github.com/chevah/styleguide/pull/47#discussion_r109395259 though the point of this section is more 'Phishing and Social Engineering' related rather than travel